Create Apply-for-the-current-round-of-Carpentries-Maintainer-Onboardi… (#1584)
carpentries/carpentries.org: _posts/2023/02/Apply-for-the-current-round-of-Carpentries-Maintainer-Onboarding.md

---
layout: page
authors: ["Vini Salazar", "Erin Becker"]

> teaser: "The Carpentries is preparing to onboard the next round of Maintainers. Applications due March 1st"

Reviewer: ```suggestion
teaser: "The Carpentries is preparing to onboard the next cohort of Maintainers. Applications due March 1st."
```

title: "Apply for the current round of Carpentries Maintainer Onboarding!"
date: 2023-02-15
time: "00:00:00"
tags: ["Community", "Maintainers"]
---

> The Carpentries is preparing to onboard the next round of Maintainers.

Reviewer: ```suggestion
The Carpentries is preparing to onboard the next cohort of Maintainers.
```

[Currently](https://github.com/carpentries/maintainer-RFCs/issues/11) we have 109 Maintainers serving across 56 core lessons. To keep the community 

> healthy, we need to plan for people wanting to step down from their current positions and finding new people to step into the role.

Reviewer: ```suggestion
healthy, we need to plan for offboarding people who want to step down from their current positions and finding new people to step into the role.
```

you will have the foundation and support to learn [the skills you will need as a Maintainer](https://www.youtube.com/watch?v=uvWhSYBkZJ0). 
The Maintainer community has a dedicated Slack channel, TopicBox list, and two monthly meetings to help support you and the lesson you are working on.

We are recruiting Maintainers for all of our official lessons, including both English and Spanish language materials. Although Maintainer Onboarding will 
be held in English, we strongly encourage applications from folks who are interested in maintaining one of the four official Carpentries lessons currently 
offered in Spanish.

If you are interested in helping maintain, improve, and grow our lessons, please 
[apply to be a lesson Maintainer](https://docs.google.com/forms/d/1vqkjrEvic6TJwCAx918T-_x5zFMVwqaiDjI8HnK1wZk).

Applications may be submitted until 
[midnight of 1 March, anywhere on Earth](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230301T235959&p1=3399).

If you have any questions about the application or about being a Maintainer, please reach out to [The Carpentries team](team@carpentries.org) or to the 

> Maintainer Community Lead, Vini Salazar.

Reviewer: ```suggestion
Maintainer Community Lead, Vini Salazar.
To contact Vini, please send him a message on [The Carpentries Slack workspace](https://swc-slack-invite.herokuapp.com/).
```
